### PR TITLE
Started release notes for 2.1.2

### PR DESCRIPTION
**Pull request by nacx**
Work started for the release notes for 2.1.2.

Things to be done:

* [x] Fix the release date to the real one once we have it.
* [x] Complete the TODOs (@gaul you've been more involved in the issues in this release. Mind having a look at the summary, known issues and contributors?).
* [ ] Once the announcement is out, get the permalink from `lists.apache.org` and generate a short url for it on `s.apache.org` using this ID: `https://s.apache.org/jclouds212` (the page already assumes that).

### CODE REVIEW

**Comment by jclouds-commentator** [comment]
Go to http://c0cdaa4e89863adf0298-fc03ca8d3e40301332464351c9025d6b.r87.cf5.rackcdn.com/ to review your changes.

**Comment by jclouds-commentator** [comment]
Go to http://6ecf81c31265b27d35de-5cc7b1fe21c4d38adc53d5a1d1daccae.r82.cf5.rackcdn.com/ to review your changes.

**Comment by gaul** [comment]
Does this automatically deploy?  Installing Jekyll 1.5.1 on my computer seems like an adventure...

**Comment by nacx** [comment]
No, it doesn't. I'll deploy it now.

**Comment by nacx** [comment]
> Installing Jekyll 1.5.1 on my computer seems like an adventure...

Installing any Ruby gem kinda is...

**Comment by nacx** [comment]
Done!

File: releasenotes/2.1.2.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+New features in Apache jclouds 2.1.2 include:
+
+* 1&1 Baremetal servers
+* EC2 r5, t3, and x1 instance types
+* GCS REGIONAL storage class
+
+### Bugs and patches
+
+* [JCLOUDS-1330](https://issues.apache.org/jira/browse/JCLOUDS-1330) - Azure ARM orphaned networks not cleaned up when node deleted
```
Comment (demobox) [inline releasenotes/2.1.2.md]: Is this list accurate? E.g. [JCLOUDS-1330](https://issues.apache.org/jira/browse/JCLOUDS-1330) is still marked as "open", and the list of issues [from JIRA](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12314430&version=12343896) looks quite different?

Comment (demobox) [reply releasenotes/2.1.2.md]: > I accidentally included unresolved issues which marked 2.1.2 as the fix version

Ah, clear, thanks!

**Pull request merged**